drivers/abp2/abp2_saul.c

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2024 CNRS, France
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_abp2
+ * @{
+ *
+ * @file
+ * @brief       ABP2 adaption to the SAUL framework.
+ *
+ * @author      David Picard <david.picard@clermont.in2p3.fr>
+ * @}
+ */
+
+#include <string.h>
+#include <stdio.h>
+
+#include "saul.h"
+#include "abp2.h"
+
+static int read_press(const void *dev, phydat_t *press)
+{
+    abp2_raw_t rawData;
+
+    if (abp2_measure_read((abp2_t *)dev, &rawData)) {
+        return -ECANCELED;
+    }
+
+    /* abp2_pressure() returns a value in thousandths of user units.
+     * here: mbar --> ubar
```

Comment:
```suggestion
```
or
```suggestion
     * here: mbar -->  0.1 mbar
```
or 
```suggestion
     * here: mbar --> 100 ubar
```

Reply:
I don't agree. The value returned by `abp2_pressure() ` is in micro-bars, or µbar or ubar.

The conversion to 0.1 mbar (1e-04 bar) happens at the next line to fit phydat.val (int16_t).

Reply:
what is the meaning of the "here: mbar --> ubar" part of the comment?
I thought it was commenting on what is happening the next line but in that case it it would be  "here: ubar --> 0.1mbar"

this is not about the function but just about the comment

Reply:
How about that ?

```text
    /* abp2_pressure() returns a int32_t in thousandths of user units,
     * i.e. µbar in this context. We need to let it fit an int16_t to be able to
     * use phydat_t.
     */
```

Reply:
lets talk about the complete comment ...